ObieWalker/EventsManager
# feature(make booking): book a center

## What does this PR do?
This PR implements a feature to create an event on the front end of the application.

## Description of Task to be completed?
- refactor booking component
- change event and center cards from functional to class components
- add scroll up button to page
- add edit and delete buttons to user events
- add conditions to catch booking conflicts
 
## How should this be manually tested?
* clone the repo and change into the project directory
* run npm install to install required packages
* create a .env file and add all the required environment variables
* start up the server with the command npm run start:dev
* start up the client side with the command npm run build:dev
* visit http://localhost:3000 and create an account/login
* on your dashboard, click the 'make a booking' tab to see available centers to book from.
*select your preference and submit. 

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?(if applicable)
- #157720819
